standard/docs/en/getting_started/releases_and_records.md

```diff
@@ -10,33 +10,33 @@ There are two kinds of OCDS document:
     <tr>
 <td valign="top" style="padding:10px;" markdown=1>
 
-**Releases** provide updates on each event in a contracting process, ideally in real-time. They can be used to notify users of new tenders, awards, contracts and other updates. One contracting process may have many releases.
+**Releases** provide updates on each event in a contracting process, ideally in real-time. They can be used to notify users of new tenders, awards, contracts and other updates. One contracting process can have many releases.
 
 </td>
 <td valign="top" style="padding:10px;" markdown=1>
 
-**Records** compile together all the information known about a contracting process, providing a snapshot view of its current state. They can also contain a versioned history of changes over time. There should only be one record for each contracting process, created by merging together the releases.
+**Records** compile together all the information known about a contracting process, providing a snapshot view of its current state. They can also contain a versioned history of changes over time. There ought to be only be one record for each contracting process at one time, created by merging together the releases.
```

Comment:
> There **is** only one record for each contracting process, created by merging together the releases.

I think adding 'at one time' muddies up the distinction between releases and records a bit, since there is only ever one record per contracting process which is updated each time a release is published.

Also I think we should change 'ought to' to 'is' in this case.

Finally, though not for this PR, I think we should remove 'created by merging together the releases' and replace it with 'providing an index to the releases' since only the release list is required in a record.

Reply:
I'll remove "at one time" which was meant to start to address #847, which the original wording doesn't address. We'll fix #847 in 1.2, and maybe put things that are allowed in a patch release into 1.1.5.

Reply:
We need to keep "ought" as "is" implies a fact about the world – and this is not one.

Reply:
Changed in 61f5cd6

Reply:
Okay great, I hadn't seen #847 

Happy with the new wording